sharktank/requirements.txt

```
@@ -1,12 +1,9 @@
 iree-turbine
 
 # Runtime deps.
-gguf==0.10.0
+gguf==0.11.0
```

could also leave this unpinned, so users can update if they want
```suggestion
gguf>=0.11.0
```

---

Set to `>=`